adapter/config/default_config.go

```diff
@@ -250,13 +250,16 @@ var defaultConfig = &Config{
 			GatewayGeneratorImpl: "org.wso2.carbon.apimgt.common.gateway.jwtgenerator.APIMgtGatewayJWTGeneratorImpl",
 			ClaimsExtractorImpl:  "org.wso2.carbon.apimgt.impl.token.ExtendedDefaultClaimsRetriever",
 			TokenTTL:             3600,
+			UseKidProperty:       false,
```

Comment:
Why do we need this property?

Are we doing anything based on this config?

Reply:
I guess early on we agreed upon making this property controllable since we used x5t property early on. But I guess we need this to be true bydefault (since it is a new version).
If we are going to assume that, none would benefit from having x5t then we could omit this.

Reply:
>> Why do we need this property? 

JWTgenerator from apimgt needs to be configured to use either x5t or kid so thats done with this property